velox/exec/TableScan.h

```diff
@@ -57,20 +57,19 @@ class TableScan : public SourceOperator {
   }
 
  private:
-  // Sets 'maxPreloadSplits' and 'splitPreloader' if prefetching
-  // splits is appropriate. The preloader will be applied to the
-  // 'first 'maxPreloadSplits' of the Tasks's split queue for 'this'
-  // when getting splits.
+  // Sets 'maxPreloadSplits' and 'splitPreloader' if prefetching splits is
+  // appropriate. The preloader will be applied to the 'first 'maxPreloadSplits'
+  // of the Task's split queue for 'this' when getting splits.
   void checkPreload();
 
-  // Sets 'split->dataSource' to be a Asyncsource that makes a
-  // DataSource to read 'split'. This source will be prepared in the
-  // background on the executor of the connector. If the DataSource is
-  // needed before prepare is done, it will be made when needed.
+  // Sets 'split->dataSource' to be an AsyncSource that makes a DataSource to
+  // read 'split'. This source will be prepared in the background on the
+  // executor of the connector. If the DataSource is needed before prepare is
+  // done, it will be made when needed.
   void preload(std::shared_ptr<connector::ConnectorSplit> split);
 
   // Process-wide IO wait time.
-  static std::atomic<uint64_t> ioWaitNanos_;
+  inline static std::atomic<uint64_t> ioWaitNanos_;
```

Comment:
Maybe move this into a global variable in cpp file only

Reply:
There is a static method that report this which is now only used by test but could be used by production later so keep it here.

Reply:
Will the `inline` here create multiple instances and the value is not longer process-wide?

Reply:
It is static so it should be singleton, and used it in many places.